packages/theme/nuxt.config.js

```diff
@@ -3,200 +3,208 @@ import theme from './themeConfig';
 import webpack from 'webpack';
 import * as dotenv from 'dotenv';
 import urlJoin from 'url-join';
+import getStore from './getStore';
 
 dotenv.config({ path: `./../../.env.${process.env.NODE_ENV}` });
 
 const serverConfig = {
   port: process.env.PORT || 3000,
   host: process.env.HOST || '0.0.0.0',
-  baseUrl: process.env.BASE_URL || 'http://localhost:3000'
+  baseUrl: process.env.BASE_URL || 'http://localhost:3000',
+  backendUrl: process.env.BACKEND_URL
```

Comment:
Consider adding `base` to the variable name (`backendBaseUrl`) to emphasize its potential structure. Or even `storeBaseUrl` since backend is not that self-explainatory in my opinion.

Reply:
Yeah that might be confusing since technically we also have the Nuxt backend. I would opt for `storeBaseUrl`. Changing the env variable name would be a breaking change though (`BACKEND_URL`) and it might confuse developers who are already using the integration.

@rafalcymerys What do you think?

Reply:
I was also thinking about `storeBaseUrl`. Seems like a more meaningful name for this use-case.